### PR TITLE
Learn to start using Secret Network

### PR DESCRIPTION
**Pull request by levackt**

### CODE REVIEW

File: secret/4_transactions/send_tokens.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const {
```
Comment (pacmessica) [inline secret/4_transactions/send_tokens.js:1]: is there a tutorial associated with this file? doesn't look like this code is part of tutorial 4

File: secret/5_contracts/main.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+  // A pen is the most basic tool you can think of for signing.
+  // This wraps a single keypair and allows for signing.
+  const signingPen = await Secp256k1Pen.fromMnemonic(mnemonic);
+
+  // Get the public key
+  const pubkey = encodeSecp256k1Pubkey(signingPen.pubkey);
+
+  // get the wallet address
+  const walletAddress = pubkeyToAddress(pubkey, 'secret');
```
Comment (pacmessica) [inline secret/5_contracts/main.js:43]: call this var `accAddress` to be consistent with tutorial? or vice versa

File: README.md [hunk 3]
```diff
@@ -1,9 +1,10 @@
 # Figment Learn Tutorials
 
-In this repository you will find application which we build for different
-blockchians during our Figment Learn tutorials.
+In this repository you will find applications which we built for different
+blockchains during our Figment Learn tutorials.
```
Comment (pacmessica) [inline README.md:4]: 👍

**Review by pacmessica — APPROVED** [review]
includes code for unwritten tutorials?

File: secret/4_transactions/send_tokens.js (hunk 1 above)
Comment (levackt) [reply secret/4_transactions/send_tokens.js]: Yes I just pushed it

**Comment by levackt** [comment]
> includes code for unwritten tutorials (6 & 7)?

Yes, want to wrap up the first 5 and hopefully still make the deadline with these. 
If it doesn't make the cut I'll remove those examples until the tutorials are also done.

**Comment by pacmessica** [comment]
looks like we're pushing to release, since there's tutorials 6-7 aren't available should we just remove them from this PR? @levackt

**Comment by levackt** [comment]
> looks like we're pushing to release, since there's tutorials 6-7 aren't available should we just remove them from this PR? @levackt

Sounds good. Removed, will push it back together with the new tutorials.

**Pull request merged**